api/src/main/java/eu/darkbot/api/managers/OreAPI.java

```diff
@@ -48,6 +49,16 @@ public interface OreAPI extends API.Singleton {
      */
     boolean showTrade(boolean show, @Nullable("if show == false") Station.Refinery tradePoint);
 
+    /**
+     * Retrieves the upgrade information associated with {@link UpgradePlace}
+     * Upgrade amount may be not always up-to-date
+     *
+     * @param upgradePlace the place from where should read the amount of upgraded ore
+     * @return pair of {@link Ore} and upgrade amount of {@link UpgradePlace}, or null when found nothing
+     */
+    @ApiStatus.AvailableSince("0.9.1")
```

Comment:
```suggestion
    @ApiStatus.AvailableSince("0.9.2")
```
If we already have released 0.9.1, this would only become available in, at least, 0.9.2